### PR TITLE
Enable predictions of survival time with `aorsf`

### LINKED ISSUE #301: Enable predictions with `type = "time"` for aorsf
see https://github.com/ropensci/aorsf/issues/37

### PR DESCRIPTION
**Pull request by hfrick**
closes #301

### CODE REVIEW

**Comment by hfrick** [comment]
Note to self: update this test in extratests

https://github.com/tidymodels/extratests/blob/0a07f45def1b87f2d01af1b1d8a005650e7306a8/tests/testthat/test-survival-augment.R#L57-L85

File: tests/testthat/test-rand_forest-aorsf.R [hunk 1]
```diff
@@ -29,6 +29,42 @@ test_that("model object", {
   )
 })
 
+# prediction: time --------------------------------------------------------
+
+test_that("time predictions", {
+  skip_if_not_installed("aorsf")
```
Comment (EmilHvitfeldt) [inline tests/testthat/test-rand_forest-aorsf.R:35]: turns out that the suggests version requirement is a little more loose than imports. So I was able to get into a situation where this was needed.

```suggestion
  skip_if_not_installed("aorsf", "0.1.2")
```

Inn general it doesn't matter much, but I also think it is good practice in this case

**Review by EmilHvitfeldt — COMMENTED** [review]
Looking good! so happy that aorsf was able to do time predictions!!

**Review by EmilHvitfeldt — APPROVED** [review]
Looking good! so happy that aorsf was able to do time predictions!!

File: tests/testthat/test-rand_forest-aorsf.R (hunk 1 above)
Comment (hfrick) [reply tests/testthat/test-rand_forest-aorsf.R]: Thanks!

**Pull request merged**